test/examples/ucx_hello_world.h

```diff
@@ -125,12 +204,19 @@ static int barrier(int oob_sock)
 
 static void generate_test_string(char *str, int size)
 {
+    char val;
     int i;
 
     for (i = 0; i < (size - 1); ++i) {
-        str[i] =  'A' + (i % 26);
+        val    = 'A' + (i % 26);
+        mem_type_allocators[test_mem_type].memcpy(&str[i],
```

Comment:
i'd generate a string at first and copy it in the end

Reply:
done